docs/troubleshooting/index.mdx

```diff
@@ -35,6 +37,72 @@ Alpine is missing some dependencies that [are required](https://github.com/datre
 Adding the following command to the CI should fix this: `apk add curl bash`
 This will add curl and bash commands which our install scripts require in order to run.
 
+---
+
+## Schema validation failure
+
+**TL;DR**  
+If you get the following message in your CLI: 
+```terminal
+❌  k8s schema validation error: could not find schema for [kindName]  
+``` 
+It means the schema validation check failed and there are three possible root causes:
+* [Worng Kubernetes schema version](troubleshooting#worng-kubernetes-schema-version) - you need to update the schema version on your dashboard
+* [Deprecated API versions](troubleshooting#deprecated-api-versions) - you need to fix your manifest
+* [Missing schema for a CRD](troubleshooting#missing-schema-for-a-crd) - you need to use the CRD Extractor
+
+### Worng Kubernetes schema version
+
+If Datree's schema validation version does not match your Kubernetes cluster version, it can produce this type of error.  
+To check your Kubernetes cluster version, you can run `kubectl version --short` and read your **Server Version**:  
+```terminal
+➜ kubectl version --short
+Server Version: v1.20.0 
+```  
+Your Datree's schema validation version can be found on your dashboard (under _SETTINGS_) or in your policy check summary:  
+```terminal
+(Summary)
+- Passing YAML validation: 1/1
+- Passing Kubernetes (1.20.0) schema validation: 0/1
+```  
+
+#### How to solve this?
+Update Datree's schema validation version in the dashboard to be the same Kubernetes version as your cluster, or pass the `--schema-version` flag ([read more](https://hub.datree.io/setup/schema-validation#changing-the-default-schema-version)).
```

Comment:
Update the default schema...

Reply:
Update Datree's schema validation => Update the default schema